win/packaging/extra.wxs.in

```diff
@@ -633,7 +618,7 @@
     <CustomAction Id="CreateDatabaseRollback" BinaryKey="wixca.dll" DllEntry="CreateDatabaseRollback"
         Execute="rollback" Return="check" Impersonate="no"/>
     <UI>
-      <ProgressText Action="CreateDatabase">Running mysql_install_db.exe</ProgressText>
+      <ProgressText Action="CreateDatabase">Running mariadb-install-db.exe</ProgressText>
```

Comment:
why?

Reply:
Reverted in bd5310ad45dfbcec57f696b4ada04083786f94ec